server/routes/api/documents/documents.ts

```diff
@@ -435,6 +436,59 @@ router.post(
   }
 );
 
+router.post(
+  "documents.users",
+  auth(),
+  pagination(),
+  validate(T.DocumentsUsersSchema),
+  async (ctx: APIContext<T.DocumentsUsersReq>) => {
+    const { id, query } = ctx.input.body;
+    const actor = ctx.state.auth.user;
+    const { offset, limit } = ctx.state.pagination;
+    const document = await Document.findByPk(id);
+    authorize(actor, "update", document);
```

Comment:
```suggestion
    authorize(actor, "read", document);
```

read is enough otherwise viewers won't be able to mention other users in comments

Reply:
Okay...How do I test this? Sorry, but it's not clear how to add comments. Ideally it should be an item of the toolbar that pops up upon text selection but I don't see it there. Or is it hidden behind some feature flag for now?

Reply:
Currently viewers cannot add comments on a text selection – they can add comments to the document though through the sidebar, and can reply to existing comments